runtime/indent/solidity.vim

```diff
@@ -1,9 +1,9 @@
 " Vim indent file
-" Language: 		Solidity
-" Acknowledgement: 	Based off of vim-javascript
-" Maintainer: 		Cothi (jiungdev@gmail.com)
-" Original Author: 	tomlion (https://github.com/tomlion/vim-solidity)
-" Last Changed: 	2022 Sep 27
+" Language: Solidity
+" Acknowledgement: Based off of vim-javascript
+" Maintainer: Cothi (jiungdev@gmail.com) 
+" Original author: tomlion (https://github.com/tomlion/vim-solidity)
```

Comment:
Simply using Author: here is more 'standard' and is recognised by the syntax highlighters.